google-oauth-client-jetty/src/main/java/com/google/api/client/extensions/jetty/auth/oauth2/LocalServerReceiver.java

```diff
@@ -254,7 +256,7 @@ public Builder setLandingPages(String successLandingPageUrl, String failureLandi
 
     @Override
     public void handle(
-        String target, HttpServletRequest request, HttpServletResponse response, int dispatch)
+        String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response)
```

Comment:
This looks like a change in public API, which we are loath to do. Can the old method be retained and add this one, maybe one with a delegate to the other?

Reply:
I have no idea how to to that! 
But, I figured after 4 years of there being a request to update a decade old highly vulnerable package ( [CVE-2011-4461][CVE-2017-7656][CVE-2017-7657][CVE-2017-7658][CVE-2017-9735][CVE-2019-10241][CVE-2019-10247] ) I might atleast post a PR to get some traction on the issue.

Reply:
Also: 
This is public within `class CallbackHandler` which is not public. I am guessing that this is only exposed to the LocalServerReceiver class that it is nested within. Although my knowledge of java is garbage so I may be completely wrong.

Reply:
If the class is non-public, that's OK then.

Reply:
Looks like there is one style issue I am creating. I will try and fix it. Please re-run kokoro.